e3db/config.py

```diff
@@ -12,7 +12,7 @@ class Config:
 
     DEFAULT_API_URL = "https://api.e3db.com"
 
-    def __init__(self, client_id, api_key_id, api_secret, public_key, private_key, public_signing_key, private_signing_key, client_email="", version="2", api_url=DEFAULT_API_URL):
+    def __init__(self, client_id, api_key_id, api_secret, public_key, private_key, client_email="", version="2", api_url=DEFAULT_API_URL, public_signing_key="", private_signing_key=""):
```

Comment:
I don't love this solution. For version 2 clients the signing keys are required. All we need is to verify that when loading a json config of a version 1 client that when the client is instantiated that we don't get a key error.

Reply:
What don't you love about this solution. It's backwards compatible in not changing the interface (positional argument order) for v1 clients. The verification of being able to load a v1 client config from json is critical, but doing both that and not changing this interface is imo better, but I don't see how it can be worse or not worthy of love ;-)

Reply:
I think going forward it is more confusing for users. I would rather have v1 clients get a little bit of magic and v2 clients get a regular old non named variables.

Reply:
If there's a why to make this magical for v1 clients where they wouldn't have to change any code written to initialize a client object, I'm not opposed to reverting this. If however this is going to break their interface, I think they will find it to be scary magic.